src/Slider/Slider.js

```diff
@@ -838,58 +862,25 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	downLeft: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonLeft.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	downRight: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonRight.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdLeft: function (sender, event) {
-		if (!this.$.buttonLeft.disabled) {
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdRight: function (sender, event) {
-		if (!this.$.buttonRight.disabled) {
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	configureSpotlightHoldPulse: function (sender, e) {
-		if (e.keyCode === 13) {
-			e.configureHoldPulse(this.holdConfig);
+	jumpButtonTriggered: function (sender, ev) {
+		var isValidEvent = true;
+		if (!sender.disabled) {
+			if (ev.keyCode != 13 && (ev.type == 'onSpotlightKeyDown' || ev.type == 'ondown')) {
```

Comment:
@DavidUM, when testing this, I only ever see 3 types of events, `onSpotlightKeyDown`, `down` (from mouse), and `onholdpulse`. Do we need to be handling the `ondown` event at all or event mapping it from the components to this handler?